go.mod

```diff
@@ -13,15 +13,12 @@ require (
 	github.com/onsi/ginkgo v1.12.1
 	github.com/onsi/gomega v1.10.1
 	github.com/openshift/api v0.0.0-20200602204738-768b7001fe69
-	github.com/operator-framework/operator-sdk v0.19.4
```

Comment:
Why is operator-sdk entirely missing on the right side? Is this the new way since operator-sdk v1.2?

Reply:
I think sdk 1.2 does not require the operator to use itself anymore; instead it asks for controller-runtime v0.6.3

https://v1-2-x.sdk.operatorframework.io/docs/upgrading-sdk-version/v1.2.0/#golang-based-operators--upgrade-sigsk8siocontroller-runtime-version-to-v063